### ISSUE TITLE
65c02 rev1: pulled signals

### ISSUE THREAD

**Issue by baldengineer** [baldengineer/Golden-Delicious#5]
- Pull-Up:
 - BE
 - RDY (done)
 - NMI (done)
 - IRQ (done)
 - RES? (through reset circuit?)
 -

**Comment by baldengineer** [comment]
basically duplicate of #4. 

RESET is held up by a 4k7 next to the reset button.
